marimo/__init__.py

```diff
@@ -94,7 +95,7 @@
 from marimo._plugins.stateless.tabs import tabs
 from marimo._plugins.stateless.tree import tree
 from marimo._plugins.stateless.video import video
-from marimo._runtime import output
+from marimo._runtime import copy, output
```

Comment:
Let's keep `copy` private for now, until we have more confidence in the executor and its associated APIs